source/RialtoGStreamerMSEAudioSink.cpp

```diff
@@ -521,6 +577,37 @@ static void rialto_mse_audio_sink_set_property(GObject *object, guint propId, co
         }
         break;
     }
+    case PROP_AUDIO_FADE:
+    {
+        const gchar *audioFadeStr = g_value_get_string(value);
+        double volume = kDefaultVolume;
+        uint32_t duration = kDefaultVolumeDuration;
+        int easeTypeInt = convertEaseTypeToInt(kDefaultEaseType);
+
+        if (sscanf(audioFadeStr, "%lf,%u,%d", &volume, &duration, &easeTypeInt) != 3)
+        {
+            GST_WARNING_OBJECT(object, "Failed to parse audio fade string: %s. Default values: volume=%lf, duration=%u, easeTypeInt=%d",
```

Comment:
These might not all be 'Default values' might only parse the first to.
I would say
'Continuing with values:'